lib/metanorma/cli/generator.rb

```diff
@@ -18,6 +18,14 @@ def run
           document_path.rmtree
         end
 
+        unless Cli.templates_path_access
+          UI.say(
+            "Sorry, #{dir} directory have no write access for current user\n" \
```

Comment:
`"Sorry, the current user is unable to write to the #{dir} directory.\n`

Reply:
Done